mount.go

```diff
@@ -257,7 +257,20 @@ func mountStorage(storage pb.Storage) error {
 		return err
 	}
 
-	return mount(storage.Source, storage.MountPoint, storage.Fstype, flags, options)
+	var fsType = storage.Fstype
+	if (storage.Driver == driverSCSIType || storage.Driver == driverBlkType) && strings.Contains(storage.Fstype, "bind") {
```

Comment:
As discussed  earlier, we should also check for the configuration passed from the runtime, the config that decides whether the block device needs to be bind-mounted or mounted to a directory.